<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4ST0B1N" name="FirefoxTest2" tests="2" failures="1" timestamp="28 Nov 2018 09:19:34 GMT" time="31.215" errors="0">
  <testcase name="login" time="12.744" classname="common.LoginCredentials">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="login" time="7.933" classname="common.LoginCredentials">
    <failure type="java.lang.AssertionError" message="expected [Signed in successfully.] but found [The user ID or password is incorrect, Please try again.]">
      <![CDATA[java.lang.AssertionError: expected [Signed in successfully.] but found [The user ID or password is incorrect, Please try again.]
	at common.TestBase.confirmationMessage(TestBase.java:984)
	at common.TestBase.logIn(TestBase.java:78)
	at common.LoginCredentials.login(LoginCredentials.java:12)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- FirefoxTest2 -->
